.coderabbit.yaml

```
@@ -0,0 +1,87 @@
+# CRITICAL: Full codebase scan configuration
+scan:
+  mode: full
+  include:
+    # Frontend files (React, Vue, Angular, etc.)
+    - "frontend/src/**/*"
+    - "frontend/components/**/*"
+    - "frontend/utils/**/*"
+    - "frontend/hooks/**/*"
+    - "frontend/lib/**/*"
+    - "frontend/app/**/*"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)